backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/log/ModularContractLogDAO.java

```diff
@@ -268,10 +267,23 @@ public Optional<ModularContractLogEntry> getLastModularContractLog(
 		final TableRecordReferenceSet modularRecordReference = TableRecordReferenceSet.of(TableRecordReference.of(I_C_OrderLine.Table_Name, orderLineId));
 
 		final Optional<I_ModCntr_Log> modCntrLog = lastRecord(ModularContractLogQuery.builder()
-																	  .contractType(MODULAR_CONTRACT)
-																	  .flatrateTermId(modularFlatrateTermId)
-																	  .referenceSet(modularRecordReference)
-																	  .build());
+				.contractType(MODULAR_CONTRACT)
+				.flatrateTermId(modularFlatrateTermId)
+				.referenceSet(modularRecordReference)
+				.build());
 		return modCntrLog.map(this::fromRecord);
 	}
+
+	public void setICProcessed(@NonNull final ModularContractLogQuery query, @NonNull final InvoiceCandidateId invoiceCandidateId)
+	{
+		final IQuery<I_ModCntr_Log> sqlQuery = toSqlQuery(query).create();
+		sqlQuery.updateDirectly()
```

Comment:
so i assume there is no (and will be no) model interceptor that listens ModCntr_Log chnages